### ISSUE TITLE
Screen Reader  is presented with  duplicate TOC

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/tr-design#95]
While reviewing the DPUB draft note found at:
http://w3c.github.io/dpub-accessibility/

My screen reader presented the table of contents twice.
I was using Windows 7, Jaws version 17, and firefox  45.

**Comment by tripu** [comment]
Shall we make use of ARIA attributes, like `aria-live`, to tell the AT that one of the TOCs is redundant?

**Comment by thierryk** [comment]
I think the proper attribute for this would be `aria-hidden="true"` (to hide one of the 2 TOCs). And the trigger that reveals the TOC should be _hidden_ as well.

**Comment by fantasai** [comment]
There should only be one ToC! What's happening that creates two ToCs????

**Comment by marcoscaceres** [comment]
Also interested from ReSpec side. Maybe it's our bug? But need more details

**Comment by fantasai** [comment]
(This is a non-ReSpec spec, fwiw: https://www.w3.org/TR/css-flexbox-1/ If it doesn't occur on that one, it's a ReSpec bug. If it does... I have no idea why, since there is only one copy of the ToC in that document.)

**Comment by marcoscaceres** [comment]
@GeorgeKerscher can you please confirm with: 
https://www.w3.org/TR/css-flexbox-1/

**Comment by halindrome** [comment]
There are not two TOCs in the DOM from ReSpec, FWIW.

**Comment by fantasai** [comment]
Closing this as invalid since there was no response to https://github.com/w3c/tr-design/issues/95#issuecomment-219570197
